app/src/main/java/ch/epfl/sdp/drone3d/service/impl/mission/ParallelogramMappingMissionService.kt

```diff
@@ -21,10 +21,13 @@ import kotlin.math.PI
 class ParallelogramMappingMissionService @Inject constructor(val droneService: DroneService): MappingMissionService {
 
     companion object{
+        //Camera angle in radians with respect to the horizontal
+        private const val cameraAngle = 0.0 // Suppose the drone is looking down
```

Comment:
IS it normal that the camera angle and cameraPitch are init with no coherent data in regard with getCameraPitch ?

Reply:
I have no idea, it felt strange when I made the merge that there were both a camera angle and a camera pitch

Reply:
The merge is not really coherent but I will fix it